vscode-antimony/src/server/stibium/stibium/antimony.lark

```diff
@@ -12,6 +12,30 @@ species : [NUMBER] ["$"] NAME
 ARROW : "->"
     | "=>"
 
+event : [reaction_name] "at" [event_delay] [event_condition_list] [event_trigger_list] ":" event_assignment_list
+
+event_delay : sum "after"
+
+event_condition_list : expressions (LOGICAL expressions)*
+
+event_trigger_list : ("," event_trigger)*
+
+// atom has to be changed to (NUMBER | var_name)
+event_trigger : "t0" "=" BOOLEAN
+    | "priority" "=" sum
+    | "fromTrigger" "=" BOOLEAN
+    | "persistent" "=" BOOLEAN
+
+event_assignment_list : event_assignment ("," event_assignment)*
+event_assignment : var_name "=" sum
+
+BOOLEAN : "true" | "false"
+
+COMPARE : ">" | "<" | ">=" | "<=" | "==" | "="

```

Comment:
No "="; just "==".  A single "=" is assignment in Anitmony, wherever it shows up.